client/flutter/lib/api/content/content_bundle.dart

```diff
@@ -43,17 +53,46 @@ class ContentBundle {
     }
   }
 
-  YamlList get contents {
+  YamlList get contentItems {
     try {
       return yaml['contents']['items'];
     } catch (err) {
       return YamlList();
     }
   }
 
-  int _getInt(String key, {int orDefault = -1}) {
-    return yaml[key];
+  String getString(String key) {
+    try {
+      return (yaml['contents'][key]).trim();
```

Comment:
Why would we trim a string?

Reply:
Dont think this was resolved pre-merge @hspinks @patniemeyer .  Would still like to understand why we don’t leave it untrimmed.

Reply:
This is related to the fact that the Protect Yourself page is currently using plain text rather than HTML - The string from the YAML includes a trailing newline and that was breaking the layout in the current page implementation.  I decided that any YAML leading and trailing whitespace was just an artifact of the file format and so trimmed it.   If this is better addressed in the YAML somehow or if we treat everything as HTML (or trim on a case by case basis in the UI layer) then this can be removed.